coreos-devel/fero-client/files/0001-gettid-glibc-2.30.patch

```diff
@@ -0,0 +1,20 @@
+--- gentoo/grpcio-sys-0.2.3/grpc/src/core/lib/support/log_linux.c   2020-11-23 18:29:18.053038547 +0000
```

Comment:
Could you add the explanation about why this patch is needed to the patch itself instead of having it in a different file?